server/statshouse/statshouse-client.cpp

```diff
@@ -123,8 +137,7 @@ bool StatsHouseClient::init_connection() {
 }
 
 void StatsHouseClient::master_send_metrics() {
-  auto [result, len] = prepare_statshouse_stats(statshouse_stats_t{tags});
-  send_metrics(result, len);
+  prepare_statshouse_stats(statshouse_stats_t{tags});
```

Comment:
let's inline this function? It's not quite `prepare` anymore

Reply:
done